EssentialFeed/EssentialFeediOS/Feed UI/Controllers/FeedImageCellController.swift

```diff
@@ -11,11 +11,13 @@ public protocol FeedImageCellControllerDelegate {
 }
 
 public final class FeedImageCellController: FeedImageView {
+	private let onOpenComments: ()->()
```

Comment:
The codebase convention is to use `Void` instead of `()`. Please, update all occurrences such as:

```suggestion
	private let onOpenComments: () -> Void
```